libuvccamera/src/main/java/com/serenegiant/usb/USBMonitor.java

```diff
@@ -213,11 +214,11 @@ public synchronized void register() throws IllegalStateException {
                     // Starting with Build.VERSION_CODES.S, it will be required to explicitly specify the mutability of PendingIntents on creation with either (@link #FLAG_IMMUTABLE} or FLAG_MUTABLE.
                     flags = PendingIntent.FLAG_MUTABLE;
                 }
-                mPermissionIntent = PendingIntent.getBroadcast(context, 0, new Intent(ACTION_USB_PERMISSION), flags);
+                mPermissionIntent = PendingIntent.getBroadcast(context, 0, new Intent(ACTION_USB_PERMISSION).setPackage(context.getPackageName()), flags);
                 final IntentFilter filter = new IntentFilter(ACTION_USB_PERMISSION);
                 // ACTION_USB_DEVICE_ATTACHED never comes on some devices so it should not be added here
                 filter.addAction(UsbManager.ACTION_USB_DEVICE_DETACHED);
-                context.registerReceiver(mUsbReceiver, filter);
+                context.registerReceiver(mUsbReceiver, filter,4);
```

Comment:
What's this hardcoded `4`?

Reply:
It's this https://developer.android.com/reference/androidx/core/content/ContextCompat#RECEIVER_NOT_EXPORTED()

Reply:
You can set up dependency on `ContextCompact` or at least create the const val `RECEIVER_NOT_EXPORTED`

Reply:
I tried using `ContextCompact` but `ContextCompat.RECEIVER_NOT_EXPORTED` doesn't exist either. I'll just make a const val `RECEIVER_NOT_EXPORTED`